build.gradle

```diff
@@ -113,9 +113,6 @@ dependencies {
     implementation "commons-io:commons-io"
 
     testImplementation "org.jetbrains.kotlin:kotlin-test-junit5:$kotlin_version"
-    testImplementation "org.junit.jupiter:junit-jupiter-api:$junitVersion"
-    testImplementation "org.junit.jupiter:junit-jupiter-params:$junitVersion"
-    testRuntimeOnly "org.junit.jupiter:junit-jupiter-engine:$junitVersion"
```

Comment:
Why have you removed all these dependencies? `kotlin-test-junit5` does provide `junit-jupiter-api` and `junit-jupiter-engine` but it does not provide `junit-jupiter-params`

Reply:
Probably `junit-jupiter-params` is not used in this project. Compilation succeed and all tests are passed. So we don't need this dependency.